src/routes/AppContent/index.tsx

```diff
@@ -101,6 +106,22 @@ const AppContent = () => {
         }
     }, []);
 
+    useEffect(() => {
+        if (
+            isAdvertiser &&
+            isBuySellPage &&
+            ((!isDesktop &&
+                (queryString.modal === 'RadioGroupFilterModal' ||
+                    queryString.modal === 'FundsModal' ||
+                    !queryString.modal)) ||
+                isDesktop)
+        ) {
+            setShowFundsBanner(true);
+        } else {
+            setShowFundsBanner(false);
+        }
+    }, [isAdvertiser, isBuySellPage, isDesktop, location, queryString]);
+
```

Comment:
as mentioned previously in my other comment, here is where I handled showing the Banner if there is any Modal shown, this just to prevent the Banner to be shown above the Modal.